lib/jekyll-import/importers/drupal_common.rb

```diff
@@ -0,0 +1,148 @@
+require 'date'
+
+module JekyllImport
+  module Importers
+    module DrupalCommon
+      # This module provides a base for the Drupal importers (at least for 6
+      # and 7; since 8 will be a different beast). Version-specific importers
+      # will need to implement the missing methods from the Importer class.
+      #
+      # The general idea is that this importer reads a MySQL database via Sequel
+      # and creates a post file for each node it finds in the Drupal database.
+
+      module ClassMethods
+        def specify_options(c)
+          c.option 'dbname', '--dbname DB', 'Database name'
+          c.option 'user', '--user USER', 'Database user name'
+          c.option 'password', '--password PW', "Database user's password (default: '')"
+          c.option 'host', '--host HOST', 'Database host name (default: "localhost")'
```

Comment:
It might make sense to specify these defaults as constants in the class and refer to them here and in `process`.